src/main/java/firebase/sdk/dashboard/data/User.java

```diff
@@ -0,0 +1,32 @@
+package firebase.sdk.dashboard.data;
+
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import java.util.Map;
+
+@AutoValue
+public abstract class User {
+
+  public static Builder newBuilder() {
+    return new AutoValue_User.Builder();
+  }
+
+  public abstract String uid();
+
+  public abstract String email();
```

Comment:
For ensuring that all users making requests are @google.com users. But it may not be necessary since we derive the email from the unique id with the firebase admin sdk.